docker/docker-compose.translator-tls-grpc.yml

```diff
@@ -0,0 +1,49 @@
+version: "3"
+
+services:
+    # Create keys:
+    # - ./.acrakeys/acra-translator/${ACRA_CLIENT_ID}_storage
+    # - ./.acrakeys/acra-writer/${ACRA_CLIENT_ID}_storage.pub
+    acra-keymaker_writer:
+        image: "cossacklabs/acra-keymaker:${ACRA_DOCKER_IMAGE_TAG:-latest}"
+        network_mode: "none"
+        environment:
+            ACRA_MASTER_KEY: ${ACRA_MASTER_KEY:-UHZ3VUNNeTJ0SEFhbWVjNkt4eDdVYkc2WnNpUTlYa0E=}
+        volumes:
+            - ./.acrakeys:/keys
+            - ./ssl/acra-client:/ssl:ro
+        command: >-
+            --client_id=""
+            --tls_cert=/ssl/acra-client.crt
+            --generate_acrawriter_keys
+            --keys_output_dir=/keys/acra-translator
+            --keys_public_output_dir=/keys/acra-translator
```

Comment:
we can skip redundant parameter, because it provided for cases when user want to separate pub/priv keys. And allow to use one parameter for equal paths.